src/Features/Core/Portable/Rename/SymbolicRenameInfo.cs

```diff
@@ -226,8 +226,9 @@ private static async Task<SyntaxToken> GetTriggerTokenAsync(Document document, i
 
                     if (sourceDocument is SourceGeneratedDocument)
                     {
-                        // The file is generated so we can't go editing it (for now)
-                        return new SymbolicRenameInfo(FeaturesResources.You_cannot_rename_this_element);
+                        // The file is generated so doesn't count towards valid spans 
+                        // we can edit.
+                        continue;
```

Comment:
I think this would work in the specific scenario from the linked issue, but could lead us to code with errors. I like giving user more flexibility here though. Is it possible to show some warning message to make sure they are aware of potential issues in the result?

Reply:
i thnik this is ok.  generally speaking the generated code will be regenerated from teh actual source symbols that *are* updated.

Reply:
In general, I think leaving generated files alone is a good stance. We don't know what will be produced, so even if they are the same symbol during the rename it doesn't mean they always will be. Code will be regenerated, and the user will get the right thing